pages/examples/components/windows.tsx

```diff
@@ -7,12 +7,113 @@ import GlobalModalManager from '@system/modals/GlobalModalManager';
+interface WindowProps extends ResizableMonospaceWindowProps{
+  key: number
+}
+
+const customParagraphStyle: React.CSSProperties = { fontFamily: 'var(--font-family-serif)', fontSize: '16px', lineHeight: '1.5', margin: 24 };
+const customButtonStyle: React.CSSProperties = { margin: '16px 8px 16px 8px' };
+const customContainerStyle: React.CSSProperties = { position: 'fixed' };
+
+const initialWindows: WindowProps[] = [
+  {
+    x: 76, y: 76, width: 248, height: 280, title: 'Window I',
+    children: (
+      <div style={customParagraphStyle}>
+        The process of growth and development almost never seems to manage to create this subtle balance between the importance of the individual parts, and the coherence of
+        the environment as a whole
+      </div>
+    ),
+    key: 0
+  },
+  {
+    x: 356, y: 76, width: 248, height: 280, title: 'Window II',
+    children: (
+      <div style={customParagraphStyle}>
+        In an organic environment, every place is unique, and the different places also cooperate, with no parts left over, to create a global whole - a whole which can be
+        identified by everyone who is part of it.
+      </div>
+    ),
+    key: 1
+  },
+  {
+    x: 634, y: 76, width: 248, height: 280, title: 'Window III',
+    children: (
+      <div style={customParagraphStyle}>
+        Making simulations of what you're going to build is tremendously useful if you can get feedback from them that will tell you where you've gone wrong and what you can do
+        about it.
+      </div>
+    ),
+    key: 2
+  },
+  {
+    x: 76, y: 394, width: 248, height: 280, title: 'Window IV',
+    children: (
+      <div style={customParagraphStyle}>
+        To work our way towards a shared language once again, we must first learn how to discover patterns which are deep, and capable of generating life.
+      </div>
+    ),
+    key: 3
+  },
+  {
+    x: 356, y: 394, width: 248, height: 280, title: 'Window V',
+    children: (
+      <div style={customParagraphStyle}>
+        From a sequence of these individual patterns, whole buildings with the character of nature will form themselves within your thoughts, as easily as sentences.
+      </div>
+    ),
+    key: 4
+  },
+  {
+    x: 634, y: 394, width: 248, height: 280, title: 'Window VI',
+    children: (
+      <div style={customParagraphStyle}>
+        When living patterns there are in a place - a room, a building, or a town - the more it comes to life as an entirety, the more it glows, the more it has that
+        self-maintaining fire which is the quality without a name.
+      </div>
+    ),
+    key: 5
+  }
+];
 
 function ExampleWindows(props) {
-  const customParagraphStyle = { fontFamily: 'var(--font-family-serif)', fontSize: '16px', lineHeight: '1.5', margin: 24 };
+  const [windows, setWindows] = React.useState<WindowProps[]>(initialWindows.map(window => ({...window, onDelete: () => closeWindow(window.key)})));
+  const [id, setId] = React.useState<number>(6);
```

Comment:
a small thing you could do to make this a little more resilient for future people making changes here is to set id as `windows.length` instead of hardcoded `6`.

currently, they would have to discover that they have to change this number if they wanted to add another initial window to the example. it's not a big deal but i can envision someone getting caught by this